src/core/common.h

```diff
@@ -11,6 +11,10 @@
 #define __STDC_LIMIT_MACROS // so we get UINT32_MAX etc
 #endif
 
+#ifdef GTA_SWITCH
+#include <switch.h>
```

Comment:
Advantages of this:
- `__SWITCH__` is defined by the switch toolchain (see https://github.com/devkitPro/pacman-packages/blob/c4c328a2ce70a8fbeb25a3833b3a0c285a5023ef/cmake/switch/Switch.cmake#L8)
- No need to add `GTA_SWITCH` in the cmake script.
Disadvantage:
- Always need to have `common.h` included when needing to check existence of `GTA_SWITCH`.
But I think this is acceptable. Or other uses should check for `__SWITCH__` instead (as they do for `_WIN32` and `__APPLE__`)

```suggestion
#ifdef __SWITCH__
#define GTA_SWITCH
#include <switch.h>
```

Reply:
Resolved by introducing new `GTA_HANDHELD` define and using `__SWITCH__` on platform specific things only.